golang/sqlboiler/templates/10_resolver.tpl

```diff
@@ -1,27 +1,48 @@
+{{- $dot := . -}}
 {{- $tableNameSingular := .Table.Name | singular -}}
 {{- $modelName := $tableNameSingular | titleCase -}}
+{{- $modelNamePlural := .Table.Name | plural | titleCase -}}
 {{- $modelNameCamel := $tableNameSingular | camelCase}}
 {{- $pkColDefs := sqlColDefinitions .Table.Columns .Table.PKey.Columns}}
 
-// {{$modelName}}sCollection is the struct representing a collection of GraphQL types
-type {{$modelName}}sCollection struct {
+// {{$modelNamePlural}}Collection is the struct representing a collection of GraphQL types
+type {{$modelNamePlural}}Collection struct {
 	nodes []{{$modelName}}
 	// future meta data goes here, e.g. count
 }
 
 // Nodes returns the list of GraphQL types
-func (c {{$modelName}}sCollection) Nodes(ctx context.Context) []{{$modelName}} {
+func (c {{$modelNamePlural}}Collection) Nodes(ctx context.Context) []{{$modelName}} {
 	return c.nodes
 }
 
-// All{{$modelName}}s retrieves {{$modelName}}s based on the provided search parameters
-func (r *Resolver) All{{$modelName}}s(ctx context.Context, args struct {
+// All{{$modelNamePlural}} retrieves {$modelNamePlural}} based on the provided search parameters
+func (r *Resolver) All{{$modelNamePlural}}(ctx context.Context, args struct {
 	Since    *graphql.ID
 	PageSize int32
 	Search *search{{$modelName}}Args
-}) ({{$modelName}}sCollection, error) {
-	result := {{$modelName}}sCollection{}
-	mods := []qm.QueryMod{qm.Limit(int(args.PageSize))}
+}) ({{$modelNamePlural}}Collection, error) {
+	result := {{$modelNamePlural}}Collection{}
+	mods := []qm.QueryMod{
+		qm.Limit(int(args.PageSize)),
+		// TODO: Add eager loading based on requested fields
+		{{/* Add eager loaders on FK relationships */}}
+		{{- range .Table.FKeys -}}
+		{{- $txt := txtsFromFKey $dot.Tables $dot.Table . -}}
+		qm.Load("{{$txt.Function.Name}}"),
```

Comment:
`qm.Load("{{template variable}}")` - Template variable is based on how the eager functions are being generated. Please see: 
https://github.com/volatiletech/sqlboiler/blob/7e0eca627f995e2476d637ada0c3b70ecb5d837c/templates/08_relationship_one_to_one_eager.tpl#L10
https://github.com/volatiletech/sqlboiler/blob/7e0eca627f995e2476d637ada0c3b70ecb5d837c/queries/eager_load.go#L68